### PR TITLE
Add the step of restarting Wazuh-dashboard when modifying logos

### PR DESCRIPTION
**Pull request by yenienserrano**
## Description

Adds the step of restarting Wazuh-dashboard when modifying the custimized logos from `opensearch-dashboards.yml`

## Closes

- https://github.com/wazuh/wazuh-documentation/issues/7667

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/user-manual/wazuh-dashboard/custom-branding.rst [hunk 1]
```diff
@@ -44,6 +44,12 @@ To customize the *global App loading* logo, do the following.
          loadingLogo:
             defaultUrl: "https://domain.org/default-logo.png"
             darkModeUrl: "https://domain.org/dark-mode-logo.png"
+   
+#. Restart the Wazuh dashboard service using this command:
```
Comment (Desvelao) [inline source/user-manual/wazuh-dashboard/custom-branding.rst:48]: **suggestion**: I would remove `using this command`.

File: source/user-manual/wazuh-dashboard/custom-branding.rst [hunk 2]
```diff
@@ -104,6 +110,12 @@ To customize the *Wazuh dashboard home* logo in the top header, do the following
             defaultUrl: "https://domain.org/default-logo.png"
             darkModeUrl: "https://domain.org/dark-mode-logo.png"
 
+#. Restart the Wazuh dashboard service using this command:
```
Comment (Desvelao) [inline source/user-manual/wazuh-dashboard/custom-branding.rst:113]: **suggestion**: I would remove `using this command`.

Comment (yenienserrano) [reply source/user-manual/wazuh-dashboard/custom-branding.rst]: Done [b1434bc](https://github.com/wazuh/wazuh-documentation/pull/7678/commits/b1434bcf4a577d3f46f65cda8905a6a06226cd2f)

File: source/user-manual/wazuh-dashboard/custom-branding.rst (hunk 1 above)
Comment (yenienserrano) [reply source/user-manual/wazuh-dashboard/custom-branding.rst]: Done [b1434bc](https://github.com/wazuh/wazuh-documentation/pull/7678/commits/b1434bcf4a577d3f46f65cda8905a6a06226cd2f)

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**